zerver/tests/test_subs.py

```diff
@@ -1022,6 +1022,39 @@ def test_api_calls(self) -> None:
         result = self.client_delete('/json/default_stream_groups/{}'.format(group_id))
         self.assert_json_error(result, "Default stream group with id '{}' does not exist.".format(group_id))
 
+    def test_invalid_default_stream_group_name(self) -> None:
+        self.login(self.example_email("iago"))
+        user_profile = self.example_user('iago')
+        realm = user_profile.realm
+
+        stream_names = ["stream1", "stream2", "stream3"]
+        group_name = ""
+        description = "This is group1"
+        streams = []
+
+        for stream_name in stream_names:
+            (stream, _) = create_stream_if_needed(realm, stream_name)
+            streams.append(stream)
+
+        result = self.client_post('/json/default_stream_groups/create',
+                                  {"group_name": group_name, "description": description,
+                                   "stream_names": ujson.dumps(stream_names)})
+        self.assert_json_error(result, "Invalid default stream group name '{}'".format(group_name))
+
+        group_name = "Prologue to the omen coming on, Have heaven and earth together demonstrated"
```

Comment:
It's better to use `'x' * 100` to make a thing of a given length; it's more readable.